src/distrs/binom.jl

```diff
@@ -11,3 +11,21 @@ import .RFunctions:
     binominvccdf,
     binominvlogcdf,
     binominvlogccdf
+
+# pdf for numbers with generic types
+binompdf(n::Real, p::Real, k::Number) = begin
+    # NOTE: fail to do below
+    #       prod(map(i -> (n + 1 - i) / i, 1:Int(k.))) * p^k * (1 - p)^(n - k)
+    n_choose_k = 1; i = 1;
+    while i <= k n_choose_k *= (n + 1 - i) / i; i += 1 end
```

Comment:
This can overflow for large `n` and `k`. Probably better to compute this as `exp(binomlogpdf)`